markdown/org/docs/patterns/shelly/options/raglanscooplength/en.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Raglan scoop length"
+---
+
+Controls how far out from the armhole the raglan seam starts to scoop. Larger arms and smaller chests will use slightly larger values for this option, but it can ususally be left at the default value without issue.
```

Comment:
```suggestion
Controls how far out from the armhole the raglan seam starts to scoop. Larger arms and smaller chests will use slightly larger values for this option, but it can usually be left at the default value without issue.
```